docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -137,28 +137,18 @@ You can easily run a full test by including it in a playbook, as long as the ``l
         debug:
           msg: '{{ testout }}'
 
-- Run the playbook and analyze the output: ``$ ansible-playbook ./testmod.yml``
+- Run the playbook and analyze the output: ``$ ANSIBLE_LIBRARY=./library ansible-playbook ./testmod.yml``
```

Comment:
Setting the default module path is only necessary for adhoc tasks, since it's a playbook-adjacent module.

Each plugin type has a default subdir (non-configurable), as well as config for additional paths. `library` is the default for modules (determined here https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/loader.py#L1554), so `ansible-playbook ./testmod.yml` will search ./library regardless of the config option.